src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/utils/ValidationMessages.java

```diff
@@ -16,6 +16,7 @@ private ValidationMessages() {}
     public static final String INVALID_CHARACTERS_ERROR_MESSAGE = "%s must only include letters a to z, numbers, and special characters such as hyphens, spaces and apostrophes";
     public static final String INVALID_EMAIL_ERROR_MESSAGE = "Email address is not in the correct format for %s, like name@example.com";
     public static final String DATE_NOT_IN_PAST_ERROR_MESSAGE = "%s must be in the past";
+    public static final String DATE_NOT_ON_OR_BEFORE_MUD_ERROR_MESSAGE = "%s must be on or before the entity's update statement date (%s)";
```

Comment:
I think the first date is the update statement date (filing date) and the second date in the message should be the made up to date, wording might need changing.

Reply:
First (replaced) string in the message is actually not a date but the qualified field name. Second string is the next 'made up to' date retrieved. I originally went with the same error text as specified in the web story but now think it would be better to change wording as suggested, so will do this.

Reply:
actually would be better to keep the field name in there or user will have to look through json to find the field that matched that date they entered

Reply:
Yes. I intend leaving the field name in there as the first string as that's the pattern used in all the other messages. I'm just changing the message text.